sgt-back-end/index.js

```diff
@@ -0,0 +1,155 @@
+const pg = require('pg');
+const express = require('express');
+
+const dataBase = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+      select *
+      from "grades"
+  `;
+
+  dataBase.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      if (grades) {
+        res.status(200);
+        res.send(grades);
+      }
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500);
+      res.send({ error: 'Unexpected error occured' });
+    });
+});
+
+app.use(express.json());
+
+app.post('/api/grades', (req, res) => {
+  const newGrades = req.body;
+
+  if (req.body.score <= 0 || Object.keys(newGrades).length < 2) {
```

Comment:
We don't really care how many keys there are, just that `req.body.name`, `req.body.course` and `req.body.score` are present and accounted for.